test/functional/specs/desktop/C2560.js

```diff
@@ -1,27 +1,22 @@
-/* eslint-disable no-console */
 import createNetworkLogger from "../../src/networkLogger";
 import gatewayDomain from "../../src/constants/gatewayDomain";
 import { responseStatus } from "../../src/assertions/index";
 
 const networkLogger = createNetworkLogger();
 
-fixture`Sandbox Tests`
+fixture`C2560`
   .page(gatewayDomain)
   .requestHooks(
     networkLogger.gatewayEndpointLogs,
     networkLogger.sandboxEndpointLogs
   );
 
-test.before(() => console.log("Test Started"));
-test.after(() => console.log("Test Completed"));
-
-const testDescription =
```

Comment:
The Allure test report doesn't require the testDescription separately when the test type is mentioned. For example: "Regression: Test description here."